packages/aws_iam_privesc/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=aws_iam_privesc
+pkgver=33.f5c92f9
+pkgrel=2
+pkgdesc='AWS IAM policy scanner that helps determine where privilege escalation can be achieved'
+arch=('any')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-recon' 'blackarch-exploitation' 'blackarch-automation')
+url='https://github.com/cyberqueen-meg/log4j-bypass'
+license=('MIT')
+depends=('python' 'python-requests' 'python-pycryptodome' 'python-termcolor' 'python-boto3' 'python-botocore' 'aws' )
+makedepends=('git')
+source=("git+https://github.com/cyberqueen-meg/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -dm 755 "$pkgdir/usr/share/$pkgname/"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm -rf LICENSE README.d
+
+  cp -a * "$pkgdir/usr/share/$pkgname/"
+
+  cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+cd /usr/share/$pkgname
```

Comment:
If the following works it's better not to `cd`

```
exec python3 /usr/share/$pkgname/aws_escalate.py "\$@"
```

Reply:
updated the code